docs/source/app_developers_guide/txn_submit_tutorial/_segments/batch5_encode.rst

```diff
@@ -0,0 +1,4 @@
+5. Encode the Batch(es)
+-----------------------
+
+In order to transmit one or more Batches to a validator for submission, they must be serialized, and *BatchList* is the protobuf message class for that purpose. It has one *batches* property, which should be set to one or more Batches.
```

Comment:
change the end of the sentence to "serialized in a *BatchList* protobuf message."